tests/tensorflow/save_keras_model.py

```diff
@@ -0,0 +1,74 @@
+import tensorflow as tf
+import numpy as np
+import pickle
+
+import mlflow.tensorflow
+from mlflow.utils.file_utils import TempDir
+import argparse
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("tracking_uri")
+parser.add_argument("task_type")
+parser.add_argument("save_as_type")
+parser.add_argument("--save_path")
```

Comment:
```suggestion
parser.add_argument("--tracking-uri")
parser.add_argument("--task-type")
parser.add_argument("--save-as-type")
parser.add_argument("--save-path")
```

Can we use keyword arguments?